### PR TITLE
New language: Icelandic

### PR DESCRIPTION
**Pull request by steinoro**
Partially translated `captions.js` to get started.

### CODE REVIEW

File: src/constants/captions_is.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+export default {
+  general: {
+    home: 'Heim',
+    student: 'Nemandi',
+    teacher: 'Kennari',
+    level: 'Stig',
+    levels: {
+      1: 'Introduction',
```
Comment (NorwegianKiwi) [inline src/constants/captions_is.js]: Missing translation?

File: src/constants/captions_is.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+      2: 'Byrjendur',
+      3: 'Með reynslu',
+      4: 'Sérfræðingur',
+    },
+    picture: 'Mynd af {{title}}',
+    glyphicon: 'Upplysingar fyrir {{title}}',
+  },
+  head: {
+    title: 'Kóðaklúbburinn',
+    description: 'On this page you will find tasks for people of all ages who wants to learn programming. ' +
```
Comment (NorwegianKiwi) [inline src/constants/captions_is.js]: missing translation?

File: src/constants/captions_is.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+                  'and is often used in the code club and programming subjects at school.',
+  },
+  frontpage: {
+    courses: 'Námskeið',
+    lessoncount: 'Lessons: {{count}}',
+    otherwebsitecourses: 'Námskeið á öðrum vefsíðum',
+    continueButton: 'Síðasti heimsökn',
+    teacherinfobox: {
+      header: 'Hello! You are now in teachermode',
+      changemode: 'Are you not a teacher or supervisor? Click the student / teacher button in the navigation menu ' +
```
Comment (NorwegianKiwi) [inline src/constants/captions_is.js]: Missing translation?

File: src/constants/captions_is.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+                  'to switch mode. For more information, click the plus sign below',
+      teacher: 'Kennari',
+      info1: 'Check out our filters and tutorials. On the left side you can filter on different school subjects, ' +
```
Comment (NorwegianKiwi) [inline src/constants/captions_is.js]: Missing translation?

File: src/assets/graphics/flag_is.svg [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (NorwegianKiwi) [inline src/assets/graphics/flag_is.svg]: Strange ^M character?

Comment (steinoro) [reply src/assets/graphics/flag_is.svg]: I don't see this in the file.

**Pull request merged**